lib/hop_swipe_cards.dart

```diff
@@ -417,43 +414,43 @@ typedef CurrentIndexInDisplay = void Function(
 
 class CardAnimation {
   static Animation<Alignment> frontCardAlign(
-      {double currentXLocation,
-      AnimationController controller,
-      Alignment currentAlignment,
-      Alignment baseAlign,
-      double swipeEdge,
-      bool pointLeft,
-      bool allowSwipeUpAndDown,
-      int swipedCards,
-      bool preventLeftSwipe,
-      Alignment afterSwipeAlignment}) {
+      {double? currentXLocation,
+      AnimationController? controller,
+      Alignment? currentAlignment,
+      Alignment? baseAlign,
+      double? swipeEdge,
+      bool? pointLeft,
+      bool? allowSwipeUpAndDown,
+      int? swipedCards,
+      bool? preventLeftSwipe,
+      Alignment? afterSwipeAlignment}) {
```

Comment:
I think we should reduce force unwrap (`!`). In `frontCardAlign` method, there are many `!`es. 
If you think it's ok to force unwrap them, they should be non-nullable type.

Reply:
@HeavenOSK i reduced as much as possible 🙏

Reply:
Nice! I added a comment for last one 👍